### DISCUSSION TITLE
Can use SQL SERVER on  Elsa2.0?

### DISCUSSION THREAD

**Discussion by hexianggui** [elsa-workflows/elsa-core#481] [Q&A] [answered]
Can use SQL SERVER on  Elsa2.0?
I can't visualize the table structure ,like the Elsa1.0.
Forgive me, I'm not familiar with SQLite.

**Comment by sfmskywalker** [comment, marked as answer]
Yes you can. Right now you can use any database server that is supported by Dapper, which is used by YesSQL, which Elsa 2 uses right now. But before its initial release, support will be added for EF Core & MongoDB, increasing the number of supported database servers.

SQLite is a light-weight, file-based database engine useful for small applications and easy testing. You can visualize the table structure using tools such as [DB Browser for SQLite](https://sqlitebrowser.org/dl/).
